pyfa.py

```diff
@@ -125,6 +125,10 @@ def _process_args(self, largs, rargs, values):
         if db_needs_update() is True:
             update_db()
 
+        from gui.app import PyfaApp
+
+        pyfa = PyfaApp(False)
```

Comment:
@zhaoweny curious on if there was a reason for moving this? In general, the order in which things are imported and initialized in this file are pretty important, so I just want to know so that I have an idea why

Reply:
Disregard, just saw your other comment on the other PR.

So... I didn't realize those were in eos. We have a hard requirement not to include wx in the eos module itself for various reasons, so we shouldn't be doing translations in them... Not really sure where we would put them then, let me give it some thought this weekend and I'll let you know :)